testsuites/store/src/main/resources/testcases/rdf-mt-inferencing/type/test003-in.nt

```diff
@@ -5,4 +5,6 @@
 # Relevant inferencing rules: rdfs10
 # Author: Arjohn Kampman (arjohn.kampman@aidministrator.nl)
 <http://example.org/prop> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.w3.org/2000/01/rdf-schema#ContainerMembershipProperty>.
-_:prop <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.w3.org/2000/01/rdf-schema#ContainerMembershipProperty>.
+
+# this entails that _:prop must be allowed as a predicate, but that is not legal in RDF
+#_:prop <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.w3.org/2000/01/rdf-schema#ContainerMembershipProperty>.
```

Comment:
@jeenbroekstra @ansell This one is a bit problematic. I have to store all the properties somehow. At the moment I'm storing them as IRIs, so I do casting during loading. If this should be allowed, then I need to store them as Resources and cast them during inferencing.

I have found an example in the rdf 1.1 semantics that has a blank node as object of a subPropertyOf statement. So I'll have to go with casting during inferencing option.

From: https://www.w3.org/TR/2014/REC-rdf11-mt-20140225/
```
ex:a rdfs:subPropertyOf _:b .
_:b rdfs:domain ex:c .
ex:d ex:a ex:e .
ex:d _:b ex:c . by rdfs7
ex:d rdf:type ex:c . by rdfs2
```

However this example uses _:b as a predicate, which I thought was illegal in RDF.